dbms/src/WindowFunctions/tests/gtest_first_value.cpp

```diff
@@ -25,6 +25,7 @@
 #include <memory>
 #include <optional>
 
+
```

Comment:
`// TODO support unsigned int.`
https://github.com/pingcap/tiflash/pull/7755#discussion_r1254210104
Support unsigned or improve this comment?